package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "turbo run build --include-dependencies --no-deps --scope=*@aws-amplify/*",
```

Comment:
`--include-dependencies` and `--no-deps` are deprecated in favor of `--filter`: https://turbo.build/repo/docs/reference/command-line-reference#--include-dependencies